### PR TITLE
Add the hacktoberfest blog

### PR DESCRIPTION
**Pull request by BellaXiang**
Signed-off-by: BellaXiang <42797797+BellaXiang@users.noreply.github.com>

### What problem does this PR solve?
Add the hacktoberfest blog

### What is changed and how does it work?
Add three images
Add a blog

### CODE REVIEW

**Comment by BellaXiang** [comment]
@dcalvin @cwen0 PTAL~

File: website/blog/2020-10-14-chaos-mesh-x-hacktoberfest.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+![Hacktoberfest T-shirt](/img/hacktoberfest-shirt.png)
+
+## Open source is the spirit 
+
+Chaos Mesh has always been a dedicated and firm advocate of open source from day 1. Within only 10 months since it was open-sourced on December 31st, 2019, Chaos Mesh has already received 25k GitHub stars, with 59 contributors from multiple organizations. And it was accepted as a [CNCF sandbox project](https://www.cncf.io/sandbox-projects/) in July 2020. The amazing growth of the project as well as the community could not have been possible without our shared commitment to the open-source community and spirit. 
+
+We hereby invite you to be part of us, starting from our handpicked issues with proper mentoring and assistance along your journey, which we hope you will find rewarding, inspiring, and most of all, fun. 
+How can you participate
+So we are all set up for you in Hacktoberfest - labeled [suitable issues](https://github.com/chaos-mesh/chaos-mesh/issues?q=is%3Aissue+is%3Aopen+label%3AHacktoberfest) with “Hacktoberfest”, and updated the [Contributing Guide](https://github.com/chaos-mesh/chaos-mesh/blob/master/CONTRIBUTING.md).
```
Comment (dcalvin) [inline website/blog/2020-10-14-chaos-mesh-x-hacktoberfest.md:27]: Please adjust the part here.

Comment (BellaXiang) [reply website/blog/2020-10-14-chaos-mesh-x-hacktoberfest.md]: Done

**Review by dcalvin — APPROVED** [review]
LGTM

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**